base_station/gui/src/components/ERDTask.vue

```diff
@@ -26,7 +26,7 @@
       <OdometryReading v-bind:odom="odom"/>
     </div>
     <div class="box cameras light-bg">
-      <Cameras v-bind:numCams="2" v-bind:channel="'/cameras_control'"/>
+      <Cameras v-bind:numCams="4" v-bind:mission="'ERD'" v-bind:channel="'/cameras_control'"/>
```

Comment:
do we want four ever for ERD? idt we have the bandwidth for that realistically

Reply:
Changed to 2